android/build.gradle

```diff
@@ -41,6 +41,6 @@ repositories {
 }
 
 dependencies {
-    implementation 'abbi.io:abbisdk:1.17.0'
+    implementation 'abbi.io:abbisdk:+'
```

Comment:
just to get the latest sdk everytime

Reply:
It could cause issues where an API change will break builds for all users that are using this package.
If you guys are following semantic versioning changing to 'abbi.io:abbisdk:1.+'  should be ok but + can add breaking changing which is not recommended.

Reply:
I agree but how else can someone change the version, manually edit the gradle file? isn't going to restore back when you clean, change the build?

Either way i agree with 'abbi.io:abbisdk:1.+' that will never include breaking changes because of a minor version. Breaking changes will only (if at all) be included in major version and only after long time with a @deprecate warning.